<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgGeometry</class>
 <widget class="QDialog" name="CQDlgGeometry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geometry Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Resize operation</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqKeepProp">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Keep proportions</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Bounding box size</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>X [m]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqSizeX">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>25</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Y [m]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqSizeY">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>45</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqSizeZ">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>65</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Z [m]</string>
     </property>
    </widget>
    <widget class="QToolButton" name="qqApplySize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>161</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>45</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Scaling factors</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqScaleZ">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>65</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqScaleY">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>45</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>45</y>
       <width>41</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqScaleX">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>25</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="qqApplyScale">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>141</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>411</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Frame orientation, rotation factors</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="qqGamma">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>65</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Gamma [deg]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqBeta">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>45</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqAlpha">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>25</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Beta [deg]</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqApplyEuler">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>231</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha [deg]</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Mesh info</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Triangle count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertex count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="qqVertexCnt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>45</y>
      <width>261</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>369</string>
    </property>
   </widget>
   <widget class="QLabel" name="qqTriangleCnt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>65</y>
      <width>261</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>400</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="qqShapeType">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>25</y>
      <width>251</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>pure</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqKeepProp</tabstop>
  <tabstop>qqSizeX</tabstop>
  <tabstop>qqSizeY</tabstop>
  <tabstop>qqSizeZ</tabstop>
  <tabstop>qqApplySize</tabstop>
  <tabstop>qqScaleX</tabstop>
  <tabstop>qqScaleY</tabstop>
  <tabstop>qqScaleZ</tabstop>
  <tabstop>qqApplyScale</tabstop>
  <tabstop>qqAlpha</tabstop>
  <tabstop>qqBeta</tabstop>
  <tabstop>qqGamma</tabstop>
  <tabstop>qqApplyEuler</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
